Update community meeting info in README file (#1327)
PlasmaPy/PlasmaPy: changelog/1327.doc.rst

> Updates deprecated meeting links. Deletes deprecated calendar
>  links. TO DO: update invitation links.

Reviewer: ```suggestion
Updated deprecated meeting and calendar links in :file:`README.md`.
```
A minor suggested change only to match the style of the other changelog entries.
